website/src/_template/contributing.md

```diff
@@ -32,52 +32,38 @@ Our website’s examples section is also our playground, please read the [Local
 
 ## Tests
 
+### Unit tests
+
 Unit tests are using Jest and can be run with:
 
 ```bash
 yarn run test:unit
 ```
 
-For end-to-end tests, we use [Webdriverio](http://webdriver.io). For it to run locally, you need to install a Selenium standalone server. Follow [the Webdriverio guide](https://webdriver.io/docs/selenium-standalone-service) to do so. You can also install a Selenium standalone server from NPM:
-
-```bash
-npm install selenium-standalone -g
-selenium-standalone install
-```
-
-And then launch it:
-
-```bash
-selenium-standalone start
-```
+### End-to-End tests
 
-After you have installed and launched the selenium standalone server, run:
+We use [Cypress](cypress.io/) for our e2e test suite. Be sure to checkout “[Writing your first test](https://docs.cypress.io/guides/getting-started/writing-your-first-test#Add-a-test-file)” and the “[Introduction to Cypress](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress#Cypress-Can-Be-Simple-Sometimes)”.
```

Comment:
You probably need to re-trigger the website build so this is updated.

Reply:
Can do that after merge